Wobble/Platform/Linux/LinuxUtils.cs

```diff
@@ -11,9 +11,17 @@ public class LinuxUtils : Utils
     {
         public override void HighlightInFileManager(string path)
         {
-            // There isn't really a standard way of doing this on Linux, so fall back to just opening the containing
-            // folder.
-            OpenNatively(Path.GetDirectoryName(path));
+            try
+            {
+                // Try launching nautilus (GNOME's file manager) first as it can highlight a file by path.
+                Process.Start("nautilus", path);
+            }
+            catch (Win32Exception)
```

Comment:
You sure that this is the only exception that can be thrown from this? If not it would probably be better to catch all.